operator/v1/types_network.go

```diff
@@ -339,6 +344,21 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type ExportNetworkFlows struct {
+	// netFlow defines the netFlow collectors that will consume the flow data exported from OVS.
+	// It is a list of strings formatted as ip:port
+	// +optional
+	NetFlow []string `json:"netFlow,omitempty"`
```

Comment:
min/max len on the slice and on the string and ideally a regex restricting the shape if that is possible.

Reply:
I don't follow, I was expecting to parse on SDN code.
Do you mean there is a way to validate at this layer? Can you  link me on how to do regex?
It would be a list of comma separated *ip:port* .

Reply:
I'm actually looking at https://github.com/openshift/api/blob/master/network/v1/types.go and I see the "Pattern" for doing regex validation.
To be fair, doing a regex for a single CIDR is a long pattern, having a list of ip ports would be something long as well.
How about documenting this better and we do the validation in Go code?

Reply:
No need for a regexp. We have a format: https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/v1/types_jsonschema.go#L35